.gitignore

```diff
@@ -9,13 +9,15 @@
 /spec/reports/
 /tmp/
 
+.DS_Store
 .rubocop-*
 /bundler.d/*
 !/bundler.d/.keep
 /config/settings.local.yml
 /config/settings/*.local.yml
 /config/environments/*.local.yml
 /spec/manageiq
+.idea/
```

Comment:
Similar with this one.

Reply:
Same - going to address this one as well.